### PR TITLE
Vert.x 4

### PR DESCRIPTION
**Pull request by cescoffier**
Won't work anytime soon...

### CODE REVIEW

File: integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartRunIT.java [hunk 1]
```diff
@@ -84,18 +85,24 @@ public void testGradleKotlinDSL(String language) throws Exception {
...
+    /** Vert.x 4 Migration: resteasy reactive fixes */
+    @Disabled
     @ParameterizedTest
     @MethodSource("provideRunAloneExamples")
     public void testRunAloneCodestartsJava(String codestart) throws Exception {
         generateProjectRunTests("maven", "java", singletonList(codestart), Collections.emptyMap());
     }
 
+    /** Vert.x 4 Migration: resteasy reactive fixes */
+    @Disabled
```
Comment (cescoffier) [inline integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartRunIT.java:97]: I actually fixed them.

Comment (ebullient) [reply integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartRunIT.java:97]: y. verifying. will clear and move to next

**Comment by ebullient** [comment]
Remaining: 

Native security failure:
```
 2021-02-04 21:38:23,563 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (vert.x-eventloop-thread-3) HTTP Request to /web-app3?state=98b741ea-9af1-4df9-81c0-ec90f140a9ea&session_state=f8fe54be-a867-4f1c-9e69-6498a29d5558&code=d68d404c-1814-4cbe-ba5b-01995d06de53.f8fe54be-a867-4f1c-9e69-6498a29d5558.6a76ced7-38ab-4487-b31b-b7a2b0905ce6 failed, error id: ff184306-146b-4441-a5c4-2b285452b5a4-3: io.quarkus.security.AuthenticationCompletionException
	at io.quarkus.oidc.runtime.CodeAuthenticationMechanism.performCodeFlow(CodeAuthenticationMechanism.java:284)
	at io.quarkus.oidc.runtime.CodeAuthenticationMechanism.access$100(CodeAuthenticationMechanism.java:51)
	at io.quarkus.oidc.runtime.CodeAuthenticationMechanism$3.apply(CodeAuthenticationMechanism.java:113)
	at io.quarkus.oidc.runtime.CodeAuthenticationMechanism$3.apply(CodeAuthenticationMechanism.java:110)
```

Native Hibernate Reactive Panache:
```
 2021-02-04 21:08:19,504 ERROR [org.hib.rea.errors] (vert.x-eventloop-thread-3) failed to execute statement [select person0_.id as id1_15_, person0_.address_id as address_5_15_, person0_.name as name2_15_, person0_.status as status3_15_, person0_.uniqueName as uniquena4_15_ from Person2 person0_ where person0_.status = 'LIVING' and person0_.name = $1 order by person0_.id]
2021-02-04 21:08:19,504 ERROR [org.hib.rea.errors] (vert.x-eventloop-thread-3) could not execute query: java.util.concurrent.CompletionException: io.vertx.pgclient.PgException: { "message": "argument of LIMIT must be type bigint, not type text", "severity": "ERROR", "code": "42804", "position": "272", "file": "parse_coerce.c", "line": "1205", "routine": "coerce_to_specific_type_typmod" }
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:632)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
```

**Comment by ebullient** [comment]
I've created https://github.com/quarkusio/quarkus/pull/14868 as a Draft PR to help w/ cross-team work resolving issues.

**Comment by cescoffier** [comment]
Closing - The 2.x branch has been pushed.

**Pull request closed without merging**